client/js/components/statement/originalStatementsTable/OriginalStatementsTableItem.vue

```diff
@@ -255,8 +255,8 @@ export default {
       if (hasOwnProp(this.statement, 'paragraph')) {
         const elementParagraphs = this.selectedElementParagraph()
 
-        if (elementParagraphs && this.statement.paragraphParentId) {
-          const paragraph = elementParagraphs.find((el) => el.id === this.statement.paragraphParentId)
+        if (elementParagraphs && this.statement.paragraphParentId && this.statement.elementId) {
```

Comment:
```suggestion
        if (elementParagraphs && this.statement.elementId) {
```

Reply:
it is tricky here, i thought we don't need it too, but actualy we do. the statement must have a paragraph otherwise we don't have to do this. 'paragraphParentId' is not used to compare here but to make sure that the statement has paragraphs